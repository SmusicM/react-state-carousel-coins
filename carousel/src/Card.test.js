import { render, fireEvent } from "@testing-library/react";
import Card from "./Card.js";
import TEST_IMAGES from "./_testCommon.js";

it("works smoke test",function(){
    const image = TEST_IMAGES[0]
    render(
      <Card
       caption={image.caption}
       src = {image.src}
       currNum={1}
       totalNum={TEST_IMAGES.length}
      />
    );
  })

  it("works with snapshot test",function(){
    const {asFragment} = render(<Card 
        caption="test caption" 
        src="https://fakefakefakegugyfug.com"
        currNum={1}
        totalNum={3}/>);
        expect(asFragment()).toMatchSnapshot()
  })