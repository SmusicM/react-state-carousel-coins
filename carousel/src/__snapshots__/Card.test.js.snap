// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works with snapshot test 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test caption
    </h4>
    <img
      alt="test caption"
      class="Card-image"
      src="https://fakefakefakegugyfug.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
